src/main/java/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/interfaces/ChanceDrop.java

```diff
@@ -0,0 +1,13 @@
+package me.mrCookieSlime.Slimefun.Objects.SlimefunItem.interfaces;
+
+import io.github.thebusybiscuit.slimefun4.core.attributes.ItemAttribute;
+
+
+public interface ChanceDrop extends ItemAttribute {

```

Comment:
Also: Missing documentation and @author tag.
Should probably also mark this as a @FunctionalInterface too.